docs/parser.md

```diff
@@ -273,6 +273,10 @@ You should enable these features only if you are using an older version.
     Some code has different meaning in Flow and in vanilla JavaScript. For example, `foo<T>(x)` is parsed as a call expression with a type argument in Flow, but as a comparison (`foo < T > x`) accordingly to the ECMAScript specification. By default, `babel-parser` parses those ambiguous constructs as Flow types only if the file starts with a `// @flow` pragma.
     Set this option to `true` to always parse files as if `// @flow` was specified.
 
+- `typescript`
+  - `dts` (`boolean`, default `false`)
```

Comment:
Can you add a `<summary>History</summary>` table (similar to the other one we have in this file) to mention that it has been added in 7.14.0?